### DISCUSSION TITLE
Need help with ytmusicapi browser

### DISCUSSION THREAD

**Discussion by zetterus** [sigma67/ytmusicapi#673] [Q&A]
Hey, i'm following instructions and get this error 
`ytmusicapi.exceptions.YTMusicUserError: The following entries are missing in your headers: x-goog-authuser, cookie. Please try a different request (such as /browse) and make sure you are logged in.`
Headers contains X-Goog-AuthUser: 0, Cookie: "smth_here"

What am I doing wrong?

**Comment by a1exsDev** [comment]
X-Goog-AuthUser needs to be a string, not a integer.

**Comment by aksh-sush** [comment]
Actually, I’m stuck at the header retrieval part itself. Can you help me out, please?

**Reply by a1exsDev** [reply]
no clue, i'm stuck on it too
